### ISSUE TITLE
Exception formatting in handlers.py

### ISSUE THREAD

**Issue by dangusev** [sanic-org/sanic#1230]
Hey! I found a lot of code like this:
```python
response_message = ('Exception occurred while handling uri: '
                                '"%s"\n%s')
logger.error(response_message, request.url, format_exc())
```
https://github.com/channelcat/sanic/blob/master/sanic/handlers.py#L86
https://github.com/channelcat/sanic/blob/master/sanic/handlers.py#L120

The problem is that such formatting doesn't work well for error trackers like Sentry, because traceback info is passed like a plain string to logger and it's hard to parse it correctly further. Why don't just use `logger.error("error message", exc_info=True)` or `logger.exception("error message")`?

Or are there any special reasons for that?

Updated:
As a workaround, I can pass my own `ErrorHandler` to Sanic object as an argument, but I'm not sure that such default behavior is convenient for the most cases.

**Comment by andreymal** [comment]
**@&#8203;stale** nope

**Comment by ahopkins** [comment]
With #1937  this is moot.
